cli/args/flags.rs

```diff
@@ -906,6 +907,20 @@ fn compile_subcommand<'a>() -> Command<'a> {
   runtime_args(Command::new("compile"), true, false)
     .trailing_var_arg(true)
     .arg(script_arg().required(true))
+    .arg(
+      Arg::new("side-module")
+        .long("side-module")
+        .help("Additional module to include in the module graph")
+        .long_help(
+          "Includes an additional module in the compiled executable's module \
+    graph. Use this flag if a dynamically imported module or a web worker main \
+    module fails to load in the executable. This flag can be passed multiple \
+    times, to include multiple additional modules.",
```

Comment:
Can I do comma separated list of urls like: `--side-module=https://deno.land/std/flags/mod.ts,https://deno.land/std/version.ts`?

Reply:
Commas are valid in URLs and filenames.

Reply:
Hm, I thought we already supported comma separated lists in other flags, but maybe I'm wrong here